### DISCUSSION TITLE
Players from the HA Player Provider are shared back with HA if MA integration is installed

### DISCUSSION THREAD

**Discussion by jpbaril** [music-assistant/support#2738] [Feature requests and ideas]
### What version of Music Assistant has the issue?

2.1.1

### What version of the Home Assistant Integration have you got installed?

2024.6.2

### Have you tried everything in the Troubleshooting FAQ and reviewed the Open and Closed Issues and Discussions to resolve this yourself?

- [X] Yes

### The problem

My media players were already integrated in Home Assistant and I already had some automations for them not related to MA.
Now that I have installed MA, I find convenient to use the HA Player Provider to get a list of ready-to-be-used players.
I also have the MA integration installed in HA.
What happens is that the MA integration creates new players in HA, but these players where coming from HA in the first place through the HA Player Provider! So I end up with duplicates of all my players.
Just as MA would filter out MA players when importing players from HA Player Provider, then MA should not create in HA players that are from the HA Player Provider.

### How to reproduce

Have some players in HA, Use HA Player Provider in MA, and then Install MA integration in HA.

### Music Providers

Filesystem

### Player Providers

HA Player Provider

### What version of Home Assistant Core are your running

2024.8.0

### What type of installation are you running?

Home Assistant OS

### On what type of hardware are you running?

Generic x86-64 (e.g. Intel NUC)

**Comment by OzGav** [comment]
Moved to feature request

**Reply by jpbaril** [reply]
I hardly see how someone would want to have duplicate players in Home Assistant, and so how this ticket is a feature request and not a bug.

**Reply by OzGav** [reply]
You can easily hide them

**Reply by jpbaril** [reply]
I did and then filled a bug report to be helpful.

**Reply by OzGav** [reply]
Weird my message didn’t come through properly. What I also needed to say is that these entities are required for the announcement functionality. So I am closing this as the entities are required and as I mentioned you can hide them if required. What we would actually recommend is firstly use native MA players if possible and then hide the HA players you don’t need.

**Reply by OzGav** [reply]
Also there has been discussion in the past as to whether those entities can be attached to the original device so that might happen in the future.
